components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementService.java

```diff
@@ -125,6 +125,18 @@ void addWorkflow(Workflow workflowDTO,
     void addAssociation(String associationName, String workflowId, String eventId, String condition) throws
                                                                                                      WorkflowException;
 
+    /**
+     * List paginated Workflows of a tenant with a filter
+     *
+     * @param tenantId Tenant Id
+     * @param limit Limit
+     * @param offset Offset
+     * @param filter Filter
```

Comment:
let's align all the parameter descriptions vertically (you can easily do it with tab) and improve the description.
```suggestion
     * @param tenantId     Tenant Id
     * @param limit        Limit
     * @param offset       Offset
     * @param filter       Filter
```

Reply:
check in all other places as well.